aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -322,13 +299,104 @@ const WizardHeader = ({ activeIndex, activeStep, setActiveIndex, setOpen }) => {
-  activeStep: PropTypes.number.isRequired,
-  setActiveIndex: PropTypes.func.isRequired,
   setOpen: PropTypes.func.isRequired,
 };
 
+const StepHeader = () => {
+  const { activeIndex, activeStep } = useContext(WizardContext);
+  return (
+    <Box id="sticky-header-single-column" gap="medium" flex={false}>
+      <Box>
+        <Text color="text-strong">
+          Step {activeStep} of {steps.length}
+        </Text>
+        <Heading color="text-strong" margin="none">
+          {steps[activeIndex].title || `Step ${activeStep} Title`}
+        </Heading>
+      </Box>
+      <Text color="text-strong" size="large">
+        {steps[activeIndex].description}
+      </Text>
+    </Box>
+  );
+};
+
+const StepFooter = () => {
+  const size = useContext(ResponsiveContext);
+  const {
+    activeIndex,
+    setActiveIndex,
+    formValues,
+    setError,
+    setAttemptedAdvance,
+  } = useContext(WizardContext);
+
+  const buttonProps = {
+    fill: size === 'small' ? 'horizontal' : undefined,
+    icon: <FormNextLink />,
+    primary: true,
+    reverse: true,
+  };
+
+  return (
+    <Footer
+      border={{ side: 'top', color: 'border' }}
+      justify="end"
+      margin={size !== 'small' ? { horizontal: 'medium' } : undefined}
+      pad={
+        size !== 'small'
+          ? { vertical: 'medium' }
+          : { vertical: 'small', horizontal: 'medium' }
+      }
+      width="medium"
+      alignSelf="center"
+      responsive={false}
+    >
+      {activeIndex < steps.length - 1 && (
+        <Button
+          {...buttonProps}
+          label="Next"
+          onClick={() => {
+            // mark that the user is trying to advance, so that onChange
+            // validation will run on any errors in the future
+            setAttemptedAdvance(true);
+
+            let nextIndex = activeIndex + 1;
+            nextIndex = nextIndex <= steps.length - 1 ? nextIndex : activeIndex;
+
+            if (validation[activeIndex]) {
+              // check for errors
+              const validationRes =
+                validation[activeIndex].validator &&
+                validation[activeIndex].validator(formValues);
+              // advance to next step if successful
+              if (validationRes && validationRes.isValid)
+                setActiveIndex(nextIndex);
+              // otherwise, display error and wizard will not advance to
+              // next step
+              else {
+                setError(validationRes);
+              }
+            } else {
+              setActiveIndex(nextIndex);
+            }
+          }}
+        />
+      )}
+      {activeIndex === steps.length - 1 && (
+        <Button
+          {...buttonProps}
+          label="Finish Wizard"
+          form="validation-form-two-column"
+          type="submit"
+        />
+      )}
+    </Footer>
+  );
+};
+
 const CancellationLayer = ({ onSetOpen }) => {
+  const { setFormValues } = useContext(WizardContext);
   return (
     <Layer
```

Comment:
Layer border-radius is smaller than in Figma - can't remember if we've "hpe themed" Layer yet, but we should get consistency here. Depending on where we are at, we should fix here, Figma, or create an issue to set defaults in theme.
![Screen Shot 2020-12-03 at 10 40 19 AM](https://user-images.githubusercontent.com/1756948/101067587-f5a9c100-3554-11eb-9048-3a150d6ce2f5.png)

Reply:
Currently, we don't have access to theme the rounding of the corners. I have filed an issue here for that work: https://github.com/grommet/grommet/issues/4805